uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java

```diff
@@ -300,6 +299,71 @@ public void enrollDevice(@NonNull final String username,
         enrollDevice(username, password, false);
     }
 
+    /**
+     * Method used to complete device enrollment with a Work Profile account. By the end of this automation,
+     * the device should have work profile enabled.
+     *
+     * @param username username of the account
+     * @param password password of the account
+     */
+    public void enrollDeviceForWorkProfile(@NonNull final String username,
```

Comment:
This I believe is actually not part of the repair work, but since you've added here already we can keep it in this PR for now.

Reply:
Yes this is for Work Profile work, I'd went ahead and added it here since I was working on both at the same time at some point.